Documentation/MRTKNuGetPackage.md

```diff
@@ -0,0 +1,80 @@
+# Mixed Reality Toolkit NuGet Package
+
+Mixed Reality Toolkit (MRTK) is now available as a NuGet package on NuGet.org. There are some differences when it comes to consuming NuGet version of MRTK as opposed to a .unitypackage, read **NuGet Package Considerations** below. If any issues are encountered, file an issue using this [template](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/new?assignees=&labels=Bug,Package%20Management%20-%20NuGet&template=bug-report.md&title=).
+
+## Installing the NuGet Package
+
+Follow these instructions to add the Mixed Reality Toolkit as a NuGet package to your project.
+
+1. Download the latest [NuGetForUnity](https://github.com/GlitchEnzo/NuGetForUnity/releases/latest) .unitypackage
+1. Import the package into your Unity project, [instructions](https://docs.unity3d.com/Manual/AssetPackages.html).
+1. In the Unity menu bar, click on **NuGet > Manage NuGet Packages**.
+
+    ![Manage NuGet Packages](Images/NuGet/ManageNuGetPackages.png)
+1. In the Search box, enter `Microsoft.MixedReality.Toolkit`.
+
+    ![Manage NuGet Packages](Images/NuGet/SearchBox.png)
+1. Choose the package you would like to install:
+    - **Microsoft.MixedReality.Toolkit.Foundation** – The core package for MRTK.
```

Comment:
I'd recommend something like this:

1. Choose the MRTK core package:
    - **Microsoft.MixedReality.Toolkit.Foundation** 
1. (Optional) Choose MRTK optional packages
    - **Microsoft.MixedReality.Toolkit.Examples (recommended) ** – The package that contains all of our examples.
    - **Microsoft.MixedReality.Toolkit.Extensions** – The package that contains extensions services and/or data providers.
    - **Microsoft.MixedReality.Toolkit.Tools** – Contains some of the tooling that comes with MRTK (Build Window, etc).

Reply:
Will make the change.